addons/l10n_in_edi/tests/test_edi_json.py

```diff
@@ -101,7 +101,7 @@ def test_edi_json(self):
                 "StCesVal": 0.0, "RndOffAmt": 0.0, "TotInvVal": 1999.59
             }
         }
-        self.assertDictEqual(json_value, expected, "Indian EDI send json value is not matched")
+        # self.assertDictEqual(json_value, expected, "Indian EDI send json value is not matched")
```

Comment:
@smetl A test case is commented!!!!!! WTF!
bro why we are writing a test case then?

CC: @jco-odoo @qdp-odoo

Reply:
fix is already here https://github.com/odoo/odoo/pull/99994